client/src/components/teams/SubTeamsModal.vue

```diff
@@ -122,6 +106,7 @@ const showDialog = computed({
   <v-dialog v-model="showDialog" fullscreen transition="dialog-bottom-transition">
     <v-img src="/images/Footer-min.jpeg" width="100%" max-height="32" cover />
     <v-card>
+      {{ selectedSubteam }}
```

Comment:
Whats this for?

Reply:
sorry, it's for testing if the data is passed correctly, delete it would be okay